### DISCUSSION TITLE
抑郁症力

### DISCUSSION THREAD

**Discussion by MISAKA-736** [share121/inter-knot#5036] [委托]
中度抑郁重度焦虑力

**Comment by KELERSAMA** [comment]
抱抱
<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by Error822** [comment]
抱抱

**Comment by linzhiyao123** [comment]
抱抱<img src="https://img-static.mihoyo.com/communityweb/upload/54f6c5f6a4bf851877158b0000e636c5.png" alt="克拉拉 给你心心" width="75" >

**Comment by March-777th** [comment]
抱抱<img src="https://img-static.mihoyo.com/communityweb/upload/a370edf80a844b3abcececf055ab9cf9.png" alt="三月七_盯" width="75" >
